scripts/make_release.sh

```diff
@@ -53,6 +53,7 @@ run_go_build "${ROOT}/sdk/go/pulumi-language-go"
 
 # Copy over the language and dynamic resource providers.
 cp "${ROOT}/sdk/nodejs/dist/pulumi-resource-pulumi-nodejs" "${PUBDIR}/bin/"
+cp "${ROOT}/sdk/nodejs/dist/pulumi-analyzer-policy" "${PUBDIR}/bin/"
```

Comment:
Feels like this should mix in the runtime name as well in the fullness of time

Reply:
Yes, I agree.